modules/administration-guide/partials/proc_defining-custom-branding-values-for-che-theia.adoc

```diff
@@ -14,20 +14,20 @@ Create a `product.json` file with a new name of the product, logo, description,
 [source,json,attrs="nowrap"]
 ----
 {
-    "name": "Red Hat CodeReady Workspaces",  <1>
-    "icon": "codeready-workspaces-icon.png", <2>
+    "name": "Red Hat Dev Spaces",  <1>
+    "icon": "devspaces-icon.png", <2>
     "logo": {                                <3>
-        "dark": "codeready-workspaces-logo-light.png",
-        "light": "codeready-workspaces-logo-dark.png"
+        "dark": "devspaces-logo-light.png",
+        "light": "devspaces-logo-dark.png"
     },
     "welcome": {                             <4>
         "title": "Welcome to Your Workspace",
         "links": ["website", "documentation"]
     },
     "links": {                               <5>
         "website": {
-            "name": "Discover CodeReady Workspaces",
-            "url": "https://developers.redhat.com/products/codeready-workspaces/overview"
+            "name": "Discover Red Hat Dev Spaces",
+            "url": "https://developers.redhat.com/products/red_hat_devspaces/overview"
```

Comment:
new URL will be https://developers.redhat.com/products/openshift_dev_spaces/overview - see https://issues.redhat.com/browse/CRW-2837